sys/include/net/gnrc/netif/hdr.h

```diff
@@ -199,6 +199,33 @@ gnrc_pktsnip_t *gnrc_netif_hdr_build(uint8_t *src, uint8_t src_len, uint8_t *dst
  */
 void gnrc_netif_hdr_print(gnrc_netif_hdr_t *hdr);
 
+/* @brief   Fetch the netif header flags of a gnrc packet
+ *
+ * @param[in]   pkt     gnrc packet from whom to fetch
+ * @param[out]  flags   pointer to flags will be stored here
+ *
+ * @return              the result of fetching flags of @p pkt
+ */
+int gnrc_netif_hdr_get_flag(gnrc_pktsnip_t* pkt, uint8_t* flags);
```

Comment:
@miri64 updated, but I didn't return `uint8_t flags`, but the fetch result in `int`, since I think maybe it is nessecery to distinguish the cases of `-ENODEV` and 'flags == 0x00'; what do you think? ;-)